src/v0/destinations/snowpipe_streaming/transform.js

```diff
@@ -1 +1,26 @@
-module.exports = require('../snowflake/transform');
+const transform = require('../snowflake/transform');
+const { processWarehouseMessage } = require('../../../warehouse');
+
+const provider = 'snowpipe_streaming';
+
+function process(event) {
+  const whSchemaVersion = event.request.query.whSchemaVersion || 'v1';
+  const whStoreEvent = event.destination.Config.storeFullEvent === true;
```

Comment:
```suggestion
  const whSchemaVersion = event.request?.query?.whSchemaVersion || 'v1';
  const whStoreEvent = event.destination?.Config?.storeFullEvent === true;
```

Reply:
`Destination` and `Destination.Config` will always be present in the event.